scripts/set_compiler_env.sh

```diff
@@ -1,84 +1,54 @@
 #!/bin/sh
+set -eu
 
-# Script to configure environment variables for Go compiler
-# to allow cross compilation
-
-: ${TARGETPLATFORM=}
-: ${TARGETOS=}
-: ${TARGETARCH=}
-: ${TARGETVARIANT=}
+# Configure environment for cross-compiling.
 
-CGO_ENABLED="$(go env CGO_ENABLED)"
-GOARCH="$(go env GOARCH)"
-GOOS="$(go env GOOS)"
-GOARM="$(go env GOARM)"
-GOBIN="$(go env GOBIN)"
+: ${TARGETPLATFORM=linux/`dpkg --print-architecture`}
 
-set -eu
+TARGETOS="$(echo $TARGETPLATFORM | cut -d"/" -f1)"
+TARGETARCH="$(echo $TARGETPLATFORM | cut -d"/" -f2)"
+TARGETVARIANT="$(echo $TARGETPLATFORM | cut -d"/" -f3)"
 
-if [ ! -z "$TARGETPLATFORM" ]; then
-  TARGETOS="$(echo $TARGETPLATFORM | cut -d"/" -f1)"
-  TARGETARCH="$(echo $TARGETPLATFORM | cut -d"/" -f2)"
-  TARGETVARIANT="$(echo $TARGETPLATFORM | cut -d"/" -f3)"
+DEBIAN_ARCH=$TARGETARCH
+if [ "$TARGETARCH" = "arm" ]
```

Comment:
I'm not sure that this `if` is correct, as currently, we support only `arm/v7`. `arm64` won't go into this `if`. So, from my PoV the logic should be as simple as if `$TARGETARCH` is `arm` and `$TARGETVARIANT` is `v7`, then `DEBIAN_ARCH=armhf`.
What do you think?

Reply:
See [the previous thread](https://github.com/photoview/photoview/pull/1053#discussion_r1761715575).